spec/models/manageiq/providers/amazon/regions_spec.rb

```diff
@@ -15,4 +15,19 @@
       expect(described_class.names).not_to include('us-gov-west-1')
     end
   end
+
+  context "adds additional regions from settings" do
+    it "includes regions from additional_regions" do
+      custom_region = {
+        "my-custom-region" => {
```

Comment:
The region will probably be symbolized to keep all keys in settings.yml serialized, so we should take care of that.